lib/squadster/formations/services/notify_squad_changes.ex

```diff
@@ -3,11 +3,13 @@ defmodule Squadster.Formations.Services.NotifySquadChanges do
   import Mockery.Macro
 
   def call(changes, squad, %{id: user_id}) do
-    mockable(Squadster.Accounts.Tasks.Notify).start_link([
-      message: text(changes),
-      target: squad,
-      options: [skip: [user_id]]
-    ])
+    if changes[:class_day] || changes[:advertisment] || changes[:squad_number] do
```

Comment:
`def call(%{class_day: nil, advertisment: nil, squad_number: ni} = changes, _, _) ...`

Reply:
Try something of that